roles/inspec/defaults/main.yml

```diff
@@ -9,9 +9,9 @@ inspec:
   dependency_path: /etc/inspec/
   profiles:
     openstack_security:
-      name: inspec-openstack-security
+      name: openstack-baseline
```

Comment:
We should not change the profile name as this change will end up having 2 profiles for same thing on existing stacks.